azurerm/resource_arm_key_vault_certificate.go

```diff
@@ -549,13 +615,31 @@ func flattenKeyVaultCertificatePolicy(input *keyvault.CertificatePolicy) []inter
 		for _, usage := range *props.KeyUsage {
 			usages = append(usages, string(usage))
 		}
+
+		sanOutput := make(map[string]interface{}, 0)
+		if san := props.SubjectAlternativeNames; san != nil {
+			if emails := san.Emails; emails != nil {
+				sanOutput["emails"] = *emails
+			}
+
+			if dnsNames := san.DNSNames; dnsNames != nil {
+				sanOutput["dns_names"] = *dnsNames
+			}
+
+			if upns := san.Upns; upns != nil {
+				sanOutput["upns"] = *upns
+			}
```

Comment:
and then append to it at the end:

```suggestion
			}
			
			sanOutputs = append(sanOutputs, sanOutput)
```